pkg/ibmvpc/parser.go

```diff
@@ -516,6 +516,10 @@ func getPgwConfig(
 	return nil
 }
 
+func ignoreFIPWarning(fipName string) string {
+	return "warning: ignoring floatingIP " + fipName
```

Comment:
```suggestion
	return "warning: ignoring floating IP " + fipName + ": unsupported floating IP target."
```
We should provide the user with some clues why this happens.

Reply:
added more details